pennylane/plugins/default_qubit.py

```diff
@@ -112,6 +114,9 @@ def apply(self, operations, rotations=None, **kwargs):
                 basis_state = par[0]
                 self.apply_basis_state(basis_state, wires)
 
+            elif isinstance(operation, MultiRZ):
```

Comment:
Is it better to check against the operation name, rather than using `isinstance`? Only advantage I can think of is that it avoids the additional import

Reply:
It might also have a slight performance advantage as well.

Reply:
I'll benchmark it. But I recently learned that with python, nothing is like it seems... See "Avoid dots" here: https://wiki.python.org/moin/PythonSpeed/PerformanceTips#Avoiding_dots...

Reply:
Here it is:
```
op.name == "MultiRZ"
0.0028035s

isinstance(op, qml.MultiRZ)
0.0008188s

MultiRZ = op.MultiRZ
isinstance(op, MultriRZ)
0.0005417s
```
I guess going for isinstance is therefore better. I guess the bottleneck is that strings have to be compared character by character while the class probably has a unique id

Reply:
Oh, that's really good to know, I would have thought it otherwise! Thanks so much for checking :blush: